articles/active-directory-b2c/api-connectors-overview.md

```diff
@@ -85,7 +85,7 @@ Using Azure AD B2C, you can add your own business logic to a user journey by cal
 ![Diagram of a RESTful service claims exchange](media/api-connectors-overview/restful-service-claims-exchange.png)
 
 > [!NOTE]
-> If there is slow or no response from the RESTful service to Azure AD B2C, the timeout is 30 seconds and the retry count is two times (meaning there are 3 tries in total). Currently, you can't configure the timeout and retry count settings.
+> If there is slow or no response from the RESTful service to Azure AD B2C, the timeout is 30 seconds and the retry count is two times (meaning there are 3 tries in total). Starting in **April 2025**, timeout durations and retries will be based on an execution policy, with the longest timeout being 10 seconds. You’ll be able to update timeout durations and retries through tenant setting.
```

Comment:
@kengaderdus Can you re-word the change to remove the future product plan? The [Contributor Guide stipulates](https://review.learn.microsoft.com/en-us/help/platform/contribute-get-started-channel-guidance?branch=main#unauthorized-content) to "Only describe what is possible with the released product to avoid confusion or mistrust if future features are delayed or canceled."